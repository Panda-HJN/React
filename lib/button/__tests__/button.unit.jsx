import * as renderer from 'react-test-renderer';
import React from 'react';
import Button from '../button';
import {mount} from 'enzyme';


describe('Button 的简单测试', () => {
    it('渲染是否成功', () => {
        const json = renderer.create(<Button/>).toJSON();
        expect(json).toMatchSnapshot();
    });
});
