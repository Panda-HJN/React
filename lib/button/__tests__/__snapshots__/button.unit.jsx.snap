// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 的简单测试 渲染是否成功 1`] = `
<div>
  button
</div>
`;
